React/game/src/app/components/App/index.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+import Game from '../Screen/Game';
+
+class App extends Component {
+  render() {
+    return <Game />;
```

Comment:
this `App` can be a functional component

Reply:
👍